.azure-pipelines/pipelines.yml

```diff
@@ -0,0 +1,24 @@
+# See https://docs.microsoft.com/en-us/azure/devops/pipelines/customize-pipeline?view=azure-devops.
+trigger:
+  branches:
+    include:
+    - "main"
+
+- stage: check
```

Comment:
From azure pipeline: 
Encountered error(s) while parsing pipeline YAML:
/.azure-pipelines/pipelines.yml: (Line: 7, Col: 1, Idx: 148) - (Line: 7, Col: 2, Idx: 149): While parsing a block mapping, did not find expected key.